Data Science/readme.md

```diff
@@ -152,6 +152,10 @@ Data science is the field that utilizes scientific methods, algorithms, and syst
         <td> <a href="https://youtu.be/X2vAabgKiuM?si=ppqTTm_mui7aOtdT">Natural Language Processing and Recommender systems </a></td>
         <td>Branch of AI enabling computers to understand, interpret, and generate human language, vital for text analysis tasks and algorithms suggesting items based on user preferences, leveraging data to enhance personalized recommendations in various domains.</td>
       </tr>
+      <tr>
+      <td><a href="https://youtu.be/yZvFH7B6gKI?si=hYj4x1sAGk9TK1VY">Data Analytics</a></td>
+      <td>Data analytics converts raw data into actionable insights. It includes a range of tools, technologies, and processes used to find trends and solve problems by using data. Data analytics can shape business processes, improve decision-making, and foster business growth.</td>
+      </tr>

```

Comment:
This is a youtube video about 'What is Data Science'. This resource should not be included here. You can remove it or find another tutorial or courses about data sceience and move it to the Free courses or Paid courses if its free or not free